app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="single-cat" id="<%= dom_id category %>">
+ <div class="category-img">
+    <%= image_tag example_img(category), class: "cat-img" %>
+  </div>
+  <p class = "cat-details">
+    <strong><%= category.name %></strong>
+    <span>$ <%= category.transactions.all.sum(:amount) %> </span>
```

Comment:
> * [ ]  I think it will be really amazing if you can also display the date of creation of each category(`created_at`) transaction as seen in the design guidelines. I think this will really be interesting since the user will keep track of when a group and transaction were created. What do you think?

Hi @omarbabou, because of time, you can be considered this comment optional and work on it after your app has been approved